### DISCUSSION TITLE
欢迎大家讨论！Welcome to discuss here!

### DISCUSSION THREAD

**Discussion by NineDayCC** [NineDayCC/HeadTracker_ESP32#1] [Announcements]
这个项目的目标是打造一个人人都能DIY的无线头追，**便宜好用**。大家有什么建议或意见，都欢迎发帖留言，帮助开源社区变得更好！:tada:

The goal of this project is to create a wireless head tracker that anyone can DIY, which is **affordable** and **user-friendl**y. Any suggestions or opinions are welcome to be posted and commented, to help improve the open-source community!:tada:

**Comment by NeariX67** [comment]
Hello,
can you publish the new PCB version and the current state of code?
I want to help you support ELRS backpack without the need of an additional receiver board. I already have a little experience on communication with ELRS Backpack via ESP-now.
Since the currently available PCB (HT Nano) can't be ordered due to unavailability of the USB-C male connector, access to the new SE version would be nice.

**Reply by NineDayCC** [reply]
Hello!
Thank you for your comment! I’d really appreciate it if you could help with the ELRS backpack communication.
I’ll publish the schematic and PCB images for the SE version as soon as possible. The detailed documentation will take a bit more time.
Let me know if you need any more info!

**Reply by NeariX67** [reply]
I have seen on the ELRS discord that you have been working on that as well, just wanted to make sure that you don't have any unpushed code that could speed up the ELRS backpack support.
I think i will also need to add support for ESP32S3 and QMI8658C IMU because this is the only hardware I have accessible at the moment. I have ordered an MPU6050 for testing as well, but it may take some time to arrive. That's why I asked to get access to the new SE Version so I can order some.

**Reply by NineDayCC** [reply]
I'v updated the docs, you can check it out on the project website. 

I only updated schematics and 2D of PCB currently. Let me know if you need any other information.

I don't have any unpushed code about ELRS backpack, because I'v been working on the web configurator.

**Reply by NeariX67** [reply]
Would be nice to also get the Gerber files, BOM and Part placement file so i can order it. Should be really cheap considering the small size and single side components.
What Type-C board should I buy? I guess USB passthrough for viewing the live feed won't be possible anymore?

**Reply by NineDayCC** [reply]
I'v updated the EDA project link, as well as BOM. 

Check #10 . [https://docs.nineday.cc/projects/headtracker-esp32/en/latest/hw-guides/SE_PCBA.html](https://docs.nineday.cc/projects/headtracker-esp32/en/latest/hw-guides/SE_PCBA.html)
